rtl_tests.c

```diff
@@ -1320,8 +1316,8 @@ void test_RtlUpperString(){
     char rnd_letter;
     char rnd_letters[101];
 
-    for(int k=0; k<100; k++){ // we use XGetTickCount as a rand() replacement
-        rnd_letter = "abcdefghijklmnopqrstuvwxyz"[(int)XGetTickCount() % 26];
+    for(int k=0; k<100; k++){ // we use GetTickCount as a rand() replacement
+        rnd_letter = "abcdefghijklmnopqrstuvwxyz"[(int)GetTickCount() % 26];
```

Comment:
This is independent of this PR, but this is *bad*. The CPU runs at 733MHz, so 1.36425648 **nano**seconds per cycle.
GetTickCount is in **milli**seconds. So when this loop runs the value will probably be constant.

It's also not giving consistent values when running the test multiple times.
You should switch to a constant seed and a random function. pdclib implements `srand` and `rand`; it also implements `rand_s` but that can't be seeded, so it's not recommended here.

Reply:
Moved into issue report: https://github.com/Cxbx-Reloaded/xbox_kernel_test_suite/issues/62